src/declaration.d

```diff
@@ -1357,6 +1357,25 @@ extern (C++) class VarDeclaration : Declaration
             storage_class &= ~stc; // strip off
         }
 
+        if (storage_class & STCscope)
+        {
+            StorageClass stc = storage_class & (STCstatic | STCextern | STCmanifest | STCtls | STCgshared);
+            if (stc)
+            {
+                OutBuffer buf;
+                stcToBuffer(&buf, stc);
+                error("cannot be scope and %s", buf.peekString());
```

Comment:
In this case it doesn't make sense. You can't read "that cannot be scope and whatever" in a newspaper. In English "scope" is a noun, not an adjective.